media/js/src/timer.js

```diff
@@ -1,19 +1,28 @@
 let time = $('meta[name="time"]').attr('content');
+let submitted = $('meta[name="submitted"]').attr('content');
 
 function strLength(num) {
     return num.toString().length;
 }
 
 let minutes = Math.floor(time / 60);
 let seconds = Math.floor(time - minutes * 60);
-if (strLength(time) > 2) {
-    $('#quiz-timer').html(
-        `<b>0${minutes}:${seconds}</b> remaining to complete quiz.`);
-
+let display_secs;
```

Comment:
There is a lot happening in the global scope here - I think this should all be contained in a function - would calling it `setupTimer()` make sense? And then just call that from the global scope.

Reply:
For some reason when I put this all in a function, the time value that's defined with the meta tag is read as undefined

Reply:
Doing this will definitely be a step in the right direction and *may* resolve the undefined meta tag thing, but if it doesn't there may still be something else going on I can look into.

Reply:
This did it!